<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSI" failures="1" tests="8" name="Test" time="0.054" errors="0" timestamp="2023-03-02T20:36:52 EST">
  <testcase classname="test.day1" name="Demo" time="0.005"/>
  <testcase classname="test.day1" name="SecondDemo" time="0.001"/>
  <testcase classname="test.day2" name="day2" time="0.001"/>
  <testcase classname="test.day3" name="MobileLogin" time="0.002"/>
  <testcase classname="test.day3" name="MobileLogin" time="0.001"/>
  <testcase classname="test.day3" name="MobileLogin" time="0.001"/>
  <testcase classname="test.day3" name="login" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day3.login(day3.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="test.day3" name="loginAPI" time="0.001"/>
</testsuite> <!-- Test -->
